pkgs/tools/graphics/gmic-qt/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, fetchurl, fetchFromGitHub, cmake, ninja, pkgconfig, qt5
+, opencv, openexr, graphicsmagick, fftw, zlib, libjpeg, libtiff, libpng
+, curl, krita
+, fetchgit }:
+
+let
+  version = "2.3.6";
+
+  gmic-version = "2.3.6";
+
+in stdenv.mkDerivation rec {
+  name = "gmic-qt-${version}";
+
+  gmic-community = fetchFromGitHub {
+    owner = "dtschump";
+    repo = "gmic-community";
+    rev = "3fd528f20a2a7d651e96078c205ff21efb9cdd1a";
+    sha256 = "08d37b49qgh5d4rds7hvr5wjj4p1y8cnbidz1cyqsibq0555pwq2";
+  };
+
+  CImg = fetchgit {
+    url = "https://framagit.org/dtschump/CImg";
+    rev = "c523f0026f3b03831c0778335fe7c7661bf9a719";
+    sha256 = "13ja8immpjkm2xskddc920axq2rp4hc2sr5ghgvgy1rshc3lp8i8";
+  };
+
+  gmic_stdlib = fetchurl {
+    name = "gmic_stdlib.h";
+    url = "http://gmic.eu/gmic_stdlib236.h";
+    sha256 = "0q5g87dsn9byd2qqsa9xrsggfb9qv055s3l2gc0jrcvpx2qbza4q";
+  };
+
+  gmic = fetchFromGitHub {
+    owner = "dtschump";
+    repo = "gmic";
+    rev = "b9a6876684f40852ca39300c9d7e7d676cb81b14";
+    sha256 = "0f69r460lyfb021m7bs8s4rxa3png51cbp1izywsy3sprjd1s57p";
+  };
+
+  gmic_qt = fetchFromGitHub {
+    owner = "c-koi";
+    repo = "gmic-qt";
+    rev = "9e992cff2db418032b849458f5c9443267c7162c";
```

Comment:
Do they have release versions or git tags instead of hashes?
Do you also have a hint for future people to know what git hashes they have to use that are compatible with each other?

Reply:
yes, they use tags with version numbers. Since the tools have the same version numbers and are relaesed almost synchronously, there should be no problem.

Reply:
Ok then please use tags to fetch the source.